docs/contributing/tasks.md

```diff
@@ -42,14 +42,14 @@ npm scripts are defined in `package.json`. These trigger a number of Gulp tasks.
 - compile JavaScript to Universal Module Definition (UMD)
 - runs `npm run postbuild:package` (which will test the output is correct)
 
-**`npm run build:dist` will do the following:**
+**`npm run build:release` will do the following:**
```

Comment:
We'll soon have 3x "dist" locations:

1. Review app `src` → `dist`
2. GOV.UK Frontend npm package `src` → `dist`
3. GOV.UK Frontend GitHub release `src` → `dist`

We've agreed to keep the same directory names but rename the task for clarity